test/verify/check-system-info

```diff
@@ -380,5 +380,34 @@ class TestSystemInfo(MachineCase):
         b.enter_page("/system")
 
 
+    @skipImage("cockpit-system doesn't have the install-on-demand feature", "rhel-7-5")
+    def testPcpAvailable(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/system")
+
+        # images which have cockpit-pcp (most of them) should not show the "enable pcp" link
+        if not m.atomic_image and m.image not in ["debian-stable"]:
+            b.wait_visible("#server-pmlogger-onoff-row")
+            b.wait_not_visible("#system-information-enable-pcp")
+
+            # remove the cockpit-pcp package
+            m.execute("pkcon remove -y pcp")
+
+        if m.image in ["centos-7", "rhel-7"] or m.image.startswith("debian") or m.image.startswith("ubuntu"):
```

Comment:
This needs a comment. IIRC this is because these OSes ship an init.d script which isn't getting removed along with the package?